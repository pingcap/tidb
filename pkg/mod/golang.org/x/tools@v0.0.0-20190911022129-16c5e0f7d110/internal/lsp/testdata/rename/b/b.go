package b

var c int //@rename("int", "uint")
