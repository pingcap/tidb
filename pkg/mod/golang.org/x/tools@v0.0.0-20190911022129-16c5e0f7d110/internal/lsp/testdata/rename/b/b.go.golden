-- uint-rename --
cannot rename builtin "int"
